happi/device.py

```diff
@@ -1,25 +1,26 @@
-from __future__ import print_function
-
-###################
-# Standard Packages
-###################
+############
+# Standard #
+############
 import re
 import six
 import sys
 import logging
 
 from collections import OrderedDict
-from prettytable import PrettyTable 
+from prettytable import PrettyTable
+
+###############
+# Third Party #
+###############
```

Comment:
It's not a huge deal, but can we stop using these comment headers? It should be understood that it always goes standard imports, blank line, third party imports, blank line, module imports. I think this is in one of the pep style guides.

Reply:
Yeah. I can stop in future files.